runtime/caml/mlvalues.h

```diff
@@ -152,9 +152,52 @@ where 0 <= R <= 31 is HEADER_RESERVED_BITS, set with the
 #define Color_hd(hd) ((hd) & HEADER_COLOR_MASK)
 #define Hd_with_color(hd, color) (((hd) &~ HEADER_COLOR_MASK) | (color))
 
+/* Macro used to deactivate thread and address sanitizers on some
+   functions. */
+#define CAMLno_tsan
+#define CAMLno_asan
```

Comment:
You are moving this outside of the CAML_INTERNALS guard. Are you doing this because you expect programmers to use CAMLno_user_tsan?

Reply:
We pushed a commit that puts `CAMLno_asan` back to where it was, guarded by `CAML_INTERNALS`. Regarding `CAMLno_user_tsan`, it is used by `Hd_val` defined in `mlvalues.h`, which is not guarded by `CAML_INTERNALS`, so we are forced to expose `CAMLno_user_tsan` and `CAMLno_tsan` as well.